packages/api/src/schema/api-schema.yaml

```diff
@@ -3039,6 +3068,12 @@ paths:
         To create a new webhook, you need to make an API call with the events
         you want to listen for and the URL that will be called when those events
         occur.
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: "#/components/schemas/webhook-payload"
```

Comment:
This is not the payload used in the code: https://github.com/livepeer/livepeer-com/blob/d93c77ce4ea801b0a4a1f3443f8c5a45139b8b6e/packages/api/src/controllers/webhook.ts#L146

Maybe we don't need the 2 schemas? Perhaps just `webhook` could be used everywhere.

Reply:
yes, but `webhook` also has many other param like `userId` which user can't send on request body. I will update the payload in code as well

Reply:
If the field is `readOnly`, it cannot be sent on request payloads (correspondingly, `writeOnly` don't come in responses). That's the ideal way of setting up these schemas, but if it's not possible for some reason no problem, we can repeat the schema for request payload.